src/css/style.css

```diff
@@ -209,8 +209,10 @@ section {
 
 /* TODO Probably delete this and combine with section above */
 section.index {
-  border-radius: 2px;
-  margin: var(--margin-big);
+  border-radius: 10px;
+  margin-top: 50px;
```

Comment:
changing the margin away from a variable makes me nervous, i'd prefer the guaranteed consistency of margins by keeping that to be a variable

Reply:
other than that, I kinda like this idea you have with the border that has these sort of half radius things. That's interesting. i think if they weren't harsh black, it could work. Maybe fiddle with some of the other color variables from `:root` to see if one of those work? we could also try reducing opacity a bit to lighten them.